src/troubleshooting/general/magento-2.4.0-checkout-error-selecting-local-payments.md

```diff
@@ -12,31 +14,33 @@ This article talks about a solution for known issue in Magento during checkout w
 
 ## Issue
 
-Preconditions
+### Preconditions
 
 * Magento 2.4.0 is installed.
-* Create 1 product and 1 category.
+* Create one product and one category.
 * Configure [Braintree Payment Method](https://devdocs.magento.com/guides/v2.4/graphql/payment-methods/braintree.html) .
 
-Steps to reproduce
+### Steps to reproduce
 
 1. Navigate to the storefront.
 1. Select items to add to the cart.
 1. Proceed to checkout.
 1. Fill out the address form a with valid address.
 1. Proceed to the Review & Payments page.
 
-Expected result
+### Expected result
 
 The local payment methods should be displayed normally, without an error message.
 
-Actual result
+### Actual result
 
 The error message, " *There are currently no available payment methods. Please update your Billing Address.* " appears, but the local payment methods will still display and function correctly.
 
 ## Solution
 
-The solution is to ignore the displayed error message, and continue with payment as normal, as all local payment methods will function normally. A fix will be available in Magento version 2.4.1, which is scheduled for release in Q4 1. \#\# Related reading
+The solution is to ignore the displayed error message, and continue with payment as normal, as all local payment methods will function normally. A fix will be available in Magento version 2.4.1, which is scheduled for release in Q4 1. 
```

Comment:
```suggestion
The solution is to ignore the displayed error message, and continue with payment as normal, as all local payment methods will function normally. The fix was available starting with Adobe Commerce version 2.4.1. 
```

Reply:
Will reword, but again the use of "Adobe Commerce" implies it is for the licensed version only.